index.html

```diff
@@ -10,8 +10,13 @@
 
 <body>
     <header class="toolbar">
+        <ul class="nav-ul">
+            <li><a href="#works">Portfolio</a></li>
+            <li><a href="#about-myself">About</a></li>
+            <li><a href="#form">Contact</a></li>
```

Comment:
- [ ] [OPTIONAL] Great job addressing the suggestions :raised_hands: but I think you could add some styling for the navigation links in order to match the Figma design. I will mark this as [OPTIONAL] because it's a minor change but I strongly suggest updating the styling for the links :wink:

  ![image](https://user-images.githubusercontent.com/81522435/144493283-f45333a7-ec24-4e03-aba7-5472a170bed7.png)

Reply:
Yes you are right! I will do it! Thank you so much! I really appreciate it! :smile: